Sources/Testing/Testing.docc/TemporaryGettingStarted.md

```diff
@@ -144,3 +144,26 @@ using Xcode 16 Beta.
 See the [Test Explorer](https://www.swift.org/documentation/articles/getting-started-with-vscode-swift.html#test-explorer)
 section of
 [Getting Started with Swift in VS Code](https://www.swift.org/documentation/articles/getting-started-with-vscode-swift.html).
+
+#### Running tests for WebAssembly
+
+To run tests for WebAssembly, install a Swift SDK for WebAssembly by following
+[the instructions](https://book.swiftwasm.org/getting-started/setup-snapshot.html).
+
+Because `swift test` doesn't know what WebAssembly environment you'd like to use
+to run your tests, building tests and running them are two separate steps. To
+build tests for WebAssembly, use the following command:
+
+```sh
+swift build --swift-sdk wasm32-unknown-wasi --build-tests
+```
+
+After building tests, you can run them using a [WASI](https://wasi.dev/)-compliant
+WebAssembly runtime, such as [Wasmtime](https://wasmtime.dev/) or
+[WasmKit](https://github.com/swiftwasm/WasmKit). For example, to run tests using
+Wasmtime, use the following command (replace `{YOURPACKAGE}` with your package's
+name):
+
+```sh
+wasmtime .build/debug/{YOURPACKAGE}PackageTests.wasm
```

Comment:
If I pass additional arguments like `--filter` here, will they be visible in Swift's `CommandLine.arguments`? If so, let's make sure to call that out in a short paragraph after this code block.